docs/conf.py

```diff
@@ -66,20 +64,21 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
     'sphinx.ext.napoleon',
+    'nbsphinx',
 ]
 
 # Napoleon settings
-napoleon_google_docstring = False
-napoleon_numpy_docstring = True
-napoleon_include_init_with_doc = False
-napoleon_include_private_with_doc = False
-napoleon_include_special_with_doc = True
-napoleon_use_admonition_for_examples = False
-napoleon_use_admonition_for_notes = False
-napoleon_use_admonition_for_references = False
-napoleon_use_param = False
-napoleon_use_rtype = True
-napoleon_use_ivar = True
+# napoleon_google_docstring = False
```

Comment:
The napoleon extension is used in the last part of the documentation QENSmodels.
I'll remove the settings which were commented out because the default settings were used.